RobotInfo[] robots
# TODO distinguish teams
BallInfo ball
float32 xsize
float32 ysize
float32 goalsize
float32 edge_x
float32 edge_y

